chemprop/cli/train.py

```diff
@@ -678,6 +690,18 @@ def build_datasets(args, train_data, val_data, test_data):
             test_dset = MulticomponentDataset(test_dsets)
         else:
             test_dset = None
+        for i, train_dataset in enumerate(train_dsets):
+            column_headers, table_rows = summarize(args.task_type, train_dataset)
+            output = build_table(
+                column_headers, table_rows, f"Summary of Training Data - Component {i}"
+            )
+            logger.info(output)
+        for i, test_dataset in enumerate(test_dsets):
+            column_headers, table_rows = summarize(args.task_type, test_dataset)
+            output = build_table(
+                column_headers, table_rows, f"Summary of Test Data - Component {i}"
+            )
+            logger.info(output)
```

Comment:
If the user specifies `-vvv` without `--log` then they will get the tables printed to stdout twice. If you can think of an elegant way to prevent that, I think it would be nice. But there probably isn't and users probably should be specifying `-vvv` without `--log` anyways.

Reply:
I gave this a shot according to [this post](https://stackoverflow.com/questions/44691558/suppress-multiple-messages-with-same-content-in-python-logging-module-aka-log-co), but it still printed the tables twice.